app/assets/stylesheets/petitions/views/_shared.scss

```diff
@@ -230,17 +230,17 @@ input.back-page {
 .debate-video {
   position: relative;
   width: 100%;
-  padding-bottom: 56.25%; // 16:9
-  background-size: 100%;
   @include inline-block;
-  background-image: image-url("graphics/graphic_house-of-commons.jpg");
-  @include device-pixel-ratio() {
-    background-image: image-url("graphics/graphic_house-of-commons-2x.jpg");
-  }
   &:hover, &:focus {
     opacity: 0.9;
   }
 
+  img {
+    width: 100%;
+    height: 100%;
+    vertical-align: bottom;
+  }
+
```

Comment:
What's the vertical-align bottom for? Is it to remove whitespace due to the `<img>` being a inline element? If so then it maybe better to do `display: block` if the `<img>` is the only element inside a `<div>`.